packages/core/src/polyfills/aria-reflect.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+/*
+ * Shim for aria atribute reflection missing in Firefox
+ * https://wicg.github.io/aom/aria-reflection-explainer.html
+ * https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaLabel
+ * Patching missing types as needed
+ */
+declare global {
+  interface Element {
+    role: string | null;
+    // ARIAMixin interface in TS lib.dom.d.ts https://github.com/microsoft/TypeScript/issues/46456
+    ariaActiveDescendant: string | null;
+    ariaControls: string | null;
+    ariaLabelledBy: string | null;
+    ariaDisabled: string | null;
+    ariaExpanded: string | null;
+    ariaSelected: string | null;
+  }
+}
+
+let roleRegistered = false;
+let ariaRegistered = false;
+
+function isNode() {
+  return (globalThis as any)?.process?.env?.JEST_WORKER_ID !== undefined; // Jest and JSDom breaks on property reflection
```

Comment:
hack: Jest throws since it doesn't seem to support aria properties correctly, once the shim is no longer required for firefox we can delete it